packages/jest-runner/src/types.ts

```diff
@@ -60,3 +63,43 @@ export interface TestWatcher extends Emittery<{change: WatcherState}> {
   isInterrupted(): boolean;
   isWatchMode(): boolean;
 }
+
+abstract class BaseTestRunner {
+  readonly isSerial?: boolean;
+  abstract readonly supportsEventEmitters: boolean;
+
+  constructor(
+    protected readonly _globalConfig: Config.GlobalConfig,
+    protected readonly _context: TestRunnerContext,
```

Comment:
Here `#` does not work. It turns these props into `private`, but they must be `protected`, so that derived classes could inherit them.

Reply:
Makes sense 👍